website/content/docs/v0.11/Guides/rbac.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Role-based access control (RBAC)
+---
+
+Talos v0.11 introduced initial support for role-based access control (RBAC).
+This guide will explain what that is and how to enable it without losing access to the cluster.
+
+## RBAC in Talos
+
+Talos uses certificates to authorize users.
+The certificate subject's organization field is used to encode user roles.
+There is a set of predefined roles that allow access to different [API methods](../../reference/api/):
+
+* `os:admin` grants access to all methods;
+* `os:reader` grants access to "safe" methods (for example, that includes the ability to list files, but does not include the ability to read files content);
+* `os:etcd:backup` grants access to [`/machine.MachineService/EtcdSnapshot`](../../reference/api/#machine.EtcdSnapshotRequest) method.
+
+Roles in the current `talosconfig` can be checked with the following command (using [`yq` v4](https://github.com/mikefarah/yq)):
+
+```sh
+$ yq eval '.contexts[.context].crt' talosconfig | base64 -d | openssl x509 -noout -text
```

Comment:
That probably could be a new command like `talosctl config info`

Reply:
Did we add this command?

Reply:
It will be in the next PR, but I'm not sure it will be backported – it might be too late for that

Reply:
https://github.com/talos-systems/talos/pull/3888